Content.Client/EntryPoint.cs

```diff
@@ -96,7 +96,7 @@ public override void Init()
 
             factory.RegisterReference<ClothingComponent, ItemComponent>();
 
-            factory.Register<SpeciesUI>();
+            factory.Register<MobUI>();
```

Comment:
@clusterfack since you wrote the mob code initially, what's your opinion on renaming `Species` to `Mob`?

Personally I don't care too much although I do believe I can see why you did it.